sql_generators/glean_usage/templates/events_stream.view.sql

```diff
@@ -4,6 +4,7 @@ CREATE OR REPLACE VIEW
   `{{ project_id }}.{{ events_stream_view }}`
 AS
 SELECT
+  DATE(submission_timestamp) AS submission_date,
```

Comment:
I don't think we have any downstream consumers yet so probably better to just take this out now